### ISSUE TITLE
EthicalAds: split `.ethical-alabaster` CSS class into `.ethical-light-theme`

### ISSUE THREAD

**Issue by humitos** [readthedocs/addons#477]
We should not be using `.ethical-alabaster` for documentation tools that are not Sphinx and are not Alabaster theme. However, as they were looking better than the default style, we started used them there.

We should create a new CSS class for these cases.

* https://github.com/readthedocs/ethical-ad-client/issues/193
* https://github.com/readthedocs/addons/pull/475#discussion_r1890626404

**Comment by ericholscher** [comment]
I looked at this briefly, and all the styling seems to be specific to Alasbaster? Seems like we should probably keep this named alabaster, and not apply it to other themes? 

https://github.com/readthedocs/addons/blob/e7d1d5fb9bfd60f68cf909e199743380540836b8/src/ethicalads.css#L204-L219

**Comment by humitos** [comment]
Right, but we are using that class with those rules for other themes than alabaster. Even, for other documentation tools that are not Sphinx: https://github.com/readthedocs/addons/blob/398478ae391de895a6989694100762dca20b2335/src/ethicalads.js#L88-L191

**Comment by ericholscher** [comment]
Right, but I'm saying that it doesn't make sense to apply that class there? Is it actually doing anything, since the structure of the content isn't the same?

**Comment by humitos** [comment]
> Is it actually doing anything, since the structure of the content isn't the same?

Have you tried removing that class and showing an ad on pages from those documentation tools?

> I'm a little confused what the goal here is.

The goal is to no use a class called `-alabaster` on themes that are not alabaster. If all those themes where we are currently using `-alabaster` are light themes, the _new_ CSS class could be called `-light-theme`. On the other hand, if that's not true for all the current usages, we can use different class names.
